sdk/tests/platform/test_az_mqtt5_policy.c

```diff
@@ -284,11 +290,44 @@ static void test_az_mqtt5_policy_outbound_sub_success(void** state)
   assert_int_equal(ref_suback, 1);
 }
 
+static void test_az_mqtt5_policy_outbound_pub_no_properties_success(void** state)
+{
+  (void)state;
+  ref_puback = 0;
+  ref_recv = 0;
+  expect_msg_properties = 0;
+
+  az_mqtt5_property_bag test_mqtt5_property_bag;
+  assert_int_equal(
+      az_mqtt5_property_bag_init(&test_mqtt5_property_bag, &test_mqtt5_client, NULL), AZ_OK);
+
+  az_mqtt5_pub_data test_mqtt5_pub_data = {
+    .topic = AZ_SPAN_LITERAL_FROM_STR(TEST_MQTT_TOPIC),
+    .qos = 0,
+    .out_id = 0,
+    .payload = AZ_SPAN_LITERAL_FROM_STR(TEST_MQTT_PAYLOAD),
+    .properties = &test_mqtt5_property_bag,
+  };
+
+  assert_int_equal(az_mqtt5_outbound_pub(&test_mqtt5_client, &test_mqtt5_pub_data), AZ_OK);
+
+  int retries = TEST_MAX_RESPONSE_CHECKS;
+  while ((ref_puback == 0 || ref_recv == 0) && retries > 0)
+  {
+    assert_int_equal(az_platform_sleep_msec(TEST_RESPONSE_DELAY_MS), AZ_OK);
+    retries--;
+  }
+
+  assert_int_equal(ref_puback, 1);
+  assert_true(ref_recv > 0);
```

Comment:
I think we need `az_mqtt5_property_bag_destroy(&test_mqtt5_property_bag);` here too

Reply:
Added it, thanks for pointing that out